addons/project/models/project_task.py

```diff
@@ -290,7 +290,7 @@ def _compute_analytic_account_id(self):
         for task in self:
             task.analytic_account_id = task.project_id.analytic_account_id
 
-    @api.depends('depend_on_ids.state', 'project_id.allow_task_dependencies')
+    @api.depends('project_id', 'stage_id', 'depend_on_ids.state', 'project_id.allow_task_dependencies')
```

Comment:
I think `project_id` is not needed in the dependencies of this compute method, right? 🤔

Reply:
yep mb followed this commit
https://github.com/odoo/odoo/pull/137820/commits/de657f8881ebcb2228fc2f7cc7ccfdeabf13e9f9

Reply:
yep indeed my bad 😅